install/ansible/cleanup.sh

```diff
@@ -0,0 +1,6 @@
+find /var/lib/containerd | xargs sudo umount
```

Comment:
> I am working on a kubeadm reset like script

I agree with @Random-Liu that it would be better to just fix `kubeadm` to work on CRI based systems.

Reply:
yes all kubeadm tools are dependent on docker. So if we don't have a temporary solution. We will leave ppl trying it out in a mess. This is a complete temporary solution.

Reply:
> I am working on a kubeadm reset like script that should use crictl to kill all the pods. This does work and cleanup most of the states. I don't clean up netns because we shouldn't be messing with it without knowing if we created or the user created it. I will update the PR with that script change. That way we can also verify CRI is cleaning up all the states.

`crictl` is a good idea. Let's make it work for us, and may also be useful to kubeadm in the future. :)